### PR TITLE
fix: Show telegram tab only when handle is non-zero string

### LINKED ISSUE #169: Telegram tab should be hidden if community doesn't have a telegram configured
**Problem**
Telegram - if nothing was entered in create/edit community, it shows error message on Telegram tab

**Acceptance Criteria**
Need to hide telegram tab if community has no telegram configured

### PR DESCRIPTION
**Pull request by near-akaia-root**
Closes #169 

## Preview

Mainnet: https://near.org/root.akaia.near/widget/gigs-board.pages.Feed?nearDevGovGigsContractAccountId=devgovgigs.near

### CODE REVIEW

File: src/gigs-board/entity/community/header.jsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ const CommunityHeader = ({ activeTabTitle, handle }) => {
       title: "GitHub",
     },
 
-    ...(communityData.telegram_handle !== null
+    ...((value?.length ?? 0) > 0
```
Comment (ailisp) [inline src/gigs-board/entity/community/header.jsx:175]: Seems a typo? I don't see any `value` in this file

Comment (near-akaia-root) [reply src/gigs-board/entity/community/header.jsx]: Yes, it is, thanks

**Review by ailisp — COMMENTED** [review]
Looks good! Did you push this change to your preview?

**Comment by near-akaia-root** [comment]
> Looks good! Did you push this change to your preview?

Done

**Comment by near-akaia-root** [comment]
UPD:

## Preview

https://near.org/root.akaia.near/widget/gigs-board.pages.Feed?nearDevGovGigsContractAccountId=devgovgigs.near

**Comment by ailisp** [comment]
@near-akaia-root all featured communities look good. But they were also good on production. I also tried all communties. NFT Builders, Wallet Builders and Validators community have an invalid tg handle. Akaia community doesn't have a tg handle. Can you edit Akaia community's tg handle to be empty string (**from production ui**, becuase from your preview UI it will be replaced with null), and see whether it is hidden?

**Comment by near-akaia-root** [comment]
@ailisp done, check it here:

https://near.org/devgovgigs.near/widget/gigs-board.pages.community.telegram?handle=akaia

**Comment by ailisp** [comment]
@near-akaia-root Looks great. Thank you!

**Pull request merged**